src/pages/patientView/copyNumberAlterations/CopyNumberTableWrapper.tsx

```diff
@@ -433,7 +499,11 @@ export default class CopyNumberTableWrapper extends React.Component<
                                         .mergedDiscreteCNADataFilteredByGene
                                 }
                                 dataStore={this.props.dataStore}
-                                initialSortColumn="Annotation"
+                                initialSortColumn={
```

Comment:
lets discuss.  we want to avoid embedding custom business logic like this in application code and figure out away to accomplish a special situation with configuration